### DISCUSSION TITLE
Example sentence for the 眼中にない entry is not useful

### DISCUSSION THREAD

**Discussion by Morgawr** [Jitendex/Jitendex#32] [Errors and Corrections]
JP: 眼中にない。 (yes, it's equivalent to the entry itself)

EN: I don't exist to you.

---

On top of the fact that the example sentence itself is the entry, which doesn't tell us anything in the first place, the translation in English is also very context-dependent and not useful at all. I'd even consider it borderline wrong.

**Comment by stephenmk** [comment]
Last time I mentioned that I might be able to hunt down all these types of examples sentences. I went ahead and did that now: https://github.com/JMdictProject/JMdictIssues/issues/115
